plugins/woocommerce/tests/php/includes/admin/list-tables/class-wc-admin-list-table-orders-test.php

```
@@ -269,4 +269,41 @@ public function test_order_search_by_product_name() {
+	 * This is a regression test for https://github.com/woocommerce/woocommerce/pull/55354.
+	 */
+	public function test_search_without_post_type_in_query_does_not_trigger_warning() {
+		$GLOBALS['pagenow'] = 'edit.php'; // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
+
+		new WC_Admin_List_Table_Orders();
+
+		$warnings = array();
+		set_error_handler( // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_set_error_handler
+			function () use ( &$warnings ) {
+				// Capture PHP warnings, notices, etc.
+				$warnings[] = compact( 'errno', 'errstr' );
```

PHPMD (2.15.0)

Avoid unused local variables such as ''errno''. (undefined)

(UnusedLocalVariable)